Protocol.md

```diff
@@ -300,6 +317,173 @@ The recipient:
 
   These script pub key forms include only standard forms accepted by the wider set of deployed Bitcoin clients in the network, which increase the chances of successful propagation to miners.
 
+# Channel
+
+## Overview
+
+The lifetime of a channel is composed of its establishment, and the possibility for each party to offer the settlement or renewal of the contract within the channel, as well as to close the channel.
+
+The establishment protocol is depicted in the following [state machine](#state-machine-representation).
+
+<img src="./images/dlc_channel_establish_ta.svg">
+
+The protocol for renewal, settlement and closing of the channel is shown in the following [state machine](#state-machine-representation).
+
+<img src="./images/dlc_channel_settle_renew.svg">
```

Comment:
Am I reading it correctly that if `RenewOffer` was not accepted (`RenewOffered` state), e.g. it timed out, it will be unilateral closed?

Why did you design it that way? Is it not safe to go back to `Established` so that `RenewOffer` can be sent again?

Reply:
Same question in regards to `SettleOffer`

Reply:
I see your point. My thinking was that if your peer is unresponsive, you probably want to close the channel. It is true that these two states are a bit special in that the application of the counter party would need to get approval (or refusal) from the user before responding. But it is also not necessary to have a single timeout for all states. You could have a long one for these two, and only close if you don't get a response after it. Other solutions I can think of:
* Being able to send `RenewOffer` and `SettleOffer` while already in these states. Things become a bit tricky though, I guess we would need to add a counter to them and have the responder reference it to know which one is being accepted/rejected.
* Being able to cancel an offer. That feels a bit easier.

 > Is it not safe to go back to Established so that RenewOffer can be sent again?

I might not be understanding exactly, but if you go back to `Established` without notifying the other peer you sort of become out of sync don't you?